packages/components/src/components/form/Textarea/index.tsx

```diff
@@ -13,134 +15,83 @@ const Wrapper = styled.div`
     justify-content: flex-start;
 `;
 
-interface StyledTextareaProps extends BaseTextareaProps {
-    tooltipAction?: React.ReactNode;
-    width?: any;
-    state?: InputState;
-    monospace?: boolean;
-    borderWidth?: number;
-    borderRadius?: number;
-}
-
-const StyledTextarea = styled.textarea<StyledTextareaProps>`
-    width: ${props => (props.width ? `${props.width}px` : '100%')};
+const StyledTextarea = styled.textarea<Pick<TextareaProps, 'state' | 'width' | 'isMonospace'>>`
+    width: ${({ width }) => (width ? `${width}px` : '100%')};
     padding: 14px 16px;
     box-sizing: border-box;
-    border: solid ${props => props.borderWidth}px
-        ${props =>
-            props.state ? getStateColor(props.state, props.theme) : props.theme.STROKE_GREY};
-    border-radius: ${props => props.borderRadius}px;
+    border: solid 1px
+        ${({ state, theme }) => (state ? getStateColor(state, theme) : theme.STROKE_GREY)};
+    border-radius: 8px;
     resize: none;
     outline: none;
-    font-family: ${variables.FONT_FAMILY.TTHOVES};
-    color: ${props => getStateColor(props.state, props.theme)};
-    background: ${props => props.theme.BG_WHITE};
-    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
-    font-size: ${variables.FONT_SIZE.SMALL};
+    font-family: ${FONT_FAMILY.TTHOVES};
+    color: ${({ state, theme }) => getStateColor(state, theme)};
+    background: ${({ theme }) => theme.BG_WHITE};
+    font-weight: ${FONT_WEIGHT.MEDIUM};
+    font-size: ${FONT_SIZE.SMALL};
     white-space: pre-wrap; /* css-3 */
     white-space: -moz-pre-wrap; /* Mozilla, since 1999 */
     white-space: -pre-wrap; /* Opera 4-6 */
```

Comment:
do we even support these browsers? maybe some could be removed

Reply:
100% Removed. 

```
Chrome >= 84
Firefox >= 78
```